client/me/get-apps/index.jsx

```diff
@@ -1,82 +1,104 @@
+        </div>
+        <p><strong>{ translate( 'Linux' ) }</strong><br />
+        { translate( 'Choose your distribution' ) }</p>
+        <Button
+          href="https://apps.wordpress.com/d/linux?ref=getapps"
+        >
+          { translate( '.TAR.GZ' ) }
+        </Button>
+        <Button
+          href="https://apps.wordpress.com/d/linux-deb?ref=getapps"
+        >
+          { translate( '.DEB' ) }
+        </Button>
+      </section>
 
-				</Card>
+    </Card>
 
-				<h2 className="form-section-heading">{ this.translate( 'Mobile apps' ) }</h2>
-				<Card className="get-apps__mobile">
+    <SectionHeader label={ translate( 'Mobile apps' ) } />
+    <Card className="get-apps__mobile">
 
-					<section className="get-apps__app">
-						<div className="get-apps__app-icon">
-							<img src="/calypso/images/me/get-apps-ios.svg" alt={ this.translate( 'iOS' ) } width="96" height="96" />
-						</div>
-						<p><strong>{ this.translate( 'iOS' ) }</strong></p>
-						<a href="https://itunes.apple.com/us/app/wordpress/id335703880?mt=8"><img src="/calypso/images/me/get-apps-app-store.png" alt={ this.translate( 'Get on the iOS App Store' ) } /></a>
-					</section>
+      <section className="get-apps__app">
+        <div className="get-apps__app-icon">
+          <img src="/calypso/images/me/get-apps-ios.svg" alt={ translate( 'iOS' ) } width="96" height="96" />
+        </div>
+        <p><strong>{ translate( 'iOS' ) }</strong></p>
+        <a
+          href="https://itunes.apple.com/us/app/wordpress/id335703880?mt=8"
+        >
+          <img
+            src="/calypso/images/me/get-apps-app-store.png"
+            alt={ translate( 'Get on the iOS App Store' ) }
+          />
+        </a>
+      </section>
 
-					<section className="get-apps__app">
-						<div className="get-apps__app-icon">
-							<img src="/calypso/images/me/get-apps-android.svg" alt={ this.translate( 'Android' ) } width="96" height="96" />
-						</div>
-						<p><strong>{ this.translate( 'Android' ) }</strong></p>
-						<a href="https://play.google.com/store/apps/details?id=org.wordpress.android"><img src="/calypso/images/me/get-apps-google-play.png" alt={ this.translate( 'Get on Google Play' ) } /></a>
-					</section>
+      <section className="get-apps__app">
+        <div className="get-apps__app-icon">
+          <img src="/calypso/images/me/get-apps-android.svg" alt={ translate( 'Android' ) } width="96" height="96" />
+        </div>
+        <p><strong>{ translate( 'Android' ) }</strong></p>
+        <a
+          href="https://play.google.com/store/apps/details?id=org.wordpress.android"
+        >
+          <img
+            src="/calypso/images/me/get-apps-google-play.png"
+            alt={ translate( 'Get on Google Play' ) }
+          />
+        </a>
+      </section>
 
-				</Card>
+    </Card>
 
-			</Main>
-		);
-	}
-} );
+  </Main>
+);
+GetApps.propTypes = propTypes;
+GetApps.defaultProps = defaultProps;
```

Comment:
With `translate` I like setting a default value in case we test it.

```js
import { identity } from 'lodash';

GetApps.defaultProps = {
	translate: identity,
};
```